Feature: Validate env file and variables during build time (#232)
rootstrap/react-native-base: README.md

7. you might need to open the android folder in Android Studio and do **File** -> **Sync project with grade files**
8. Last but not least, make sure that you add the new environment to the `bump` script at

#### ENV file validation during build time

> We have in place a validation that will check if the env file exists or has all the variables defined before building the app. This is to avoid building the app with the wrong env file and having to wait for the build to fail to realize that.

Reviewer: ```suggestion
We have in place a validation that will check if the env file exists and has all the variables defined before building the app. This is to avoid building the app with the wrong env file and having to wait for the build to fail to realize that.
```
Just a suggestion

> Variables are defined `src/@types/env` and the validation is done in `scripts/validation-env.ts`.

Reviewer: ```suggestion
Variables are defined in `src/@types/env` and the validation is done in `scripts/validation-env.ts`.
```

```
  APP_ENV={dev | prod | staging | qa} react-native run-{android | ios}
```

> Note: For iOS the validation ocurred within the `scripts/simulator.sh` script. For Android it occurred as an inline command in the `package.json` scripts

Reviewer: ```suggestion
Note: For iOS the validation ocurs within the `scripts/simulator.sh` script. For Android it occurs as an inline command in the `package.json` scripts
```
